src/main.cpp

```diff
@@ -9,6 +9,142 @@
 
 namespace fs = filesystem;
 
+void ArgConfig( args::ArgumentParser &parser) {
+
+    Config config;
+
+    args::ValueFlag<int> columnlimit(parser, "column limit", "Column limit of one line", {"column-limit"});
+    args::ValueFlag<int> indentwidth(parser, "indentation width", "Number of spaces used for indentation", {"indent-width"});
+    args::ValueFlag<int> tabwidth(parser, "tab width", "Number of spaces used per tab", {"tab-width"});
+    args::ValueFlag<int> continuationindentwidth(parser, "Continuation indentation width", "Indent width for continuations line", {"continuation-indent-width"});
+    args::ValueFlag<int> spacesbeforecall(parser, "spaces before call", "Space on function calls", {"spaces-before-call"});
+    args::ValueFlag<int> columntablelimit(parser, "column table limit", "Column limit of each line of a table", {"column-table-limit"});
+    args::ValueFlag<char> tablesep(parser, "table separator", "Character to separate table fields", {"table-sep"});
+    if (tablesep) { config.ArgmapChar["table_sep"] = args::get(tablesep); }
+
+    if (columnlimit) { config.ArgmapInt["column_limit"] = args::get(columnlimit); }
+    if (indentwidth) { config.ArgmapInt["indent_width"] = args::get(indentwidth); }
+    if (tabwidth) { config.ArgmapInt["tab_width"] = args::get(tabwidth); }
+    if (continuationindentwidth) { config.ArgmapInt["continuation_indent_width"] = args::get(continuationindentwidth); }
+    if (spacesbeforecall) { config.ArgmapInt["spaces_before_call"] = args::get(spacesbeforecall); }
+    if (columntablelimit) { config.ArgmapInt["column_table_limit"] = args::get(columntablelimit); }
+
+    args::Group usetab(parser, "This group is all exclusive:", args::Group::Validators::Xor);
```

Comment:
Please review this in the entire code.
Change `usetab` to `optusetab`.
Change the options to `usetab` for true and `nousetab` for false.